go.mod

```diff
@@ -76,25 +74,4 @@ replace (
 	github.com/google/addlicense => github.com/the-mesh-for-data/addlicense v0.0.0-20200913135744-636c44b42906
 	github.com/onsi/gomega => github.com/onsi/gomega v1.10.0
 	helm.sh/helm/v3 v3.5.2 => github.com/the-mesh-for-data/helm/v3 v3.1.0-rc.1.0.20210505082940-f9c923d43d3a
-	k8s.io/api => k8s.io/api v0.19.2
```

Comment:
Do we need the replacement for the docker stuff? Why are they even there.

Reply:
> 
> 
> Do we need the replacement for the docker stuff? Why are they even there.

Because of helm dependency. I have tried to remove them - got:
```
go: helm.sh/helm/v3@v3.5.2 requires
	github.com/deislabs/oras@v0.10.0 requires
	github.com/docker/distribution@v0.0.0-00010101000000-000000000000: invalid version: unknown revision 000000000000
```